packages/opentelemetry/src/spanExporter.ts

```diff
@@ -257,6 +209,29 @@ export class SentrySpanExporter {
 
     return sentSpans;
   }
+
+  /** Remove "expired" span id entries from the _sentSpans cache. */
+  private _flushSentSpanCache(): void {
+    const currentTimestamp = Date.now();
+    // Note, it is safe to delete items from the map as we go: https://stackoverflow.com/a/35943995/90297
+    for (const [spanId, expirationTime] of this._sentSpans.entries()) {
+      if (expirationTime <= currentTimestamp) {
+        this._sentSpans.delete(spanId);
+      }
+    }
+  }
+
+  /** Check if a node is a completed root node or a node whose parent has already been sent */
+  private _nodeIsCompletedRootNode(node: SpanNode): node is SpanNodeCompleted {
```

Comment:
We may wanna change the name of this method to reflect what id actually does better from the calling side. (my bad for not changing it in the earlier patch)